.wuf/components.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
Is this auto generated or manual?

Reply:
It's generated

Reply:
So put it in gitignore. It would avoid unnecessary PR conflicts which should not change anything in the user flow